packages/cosmic-swingset/src/chain-main.js

```diff
@@ -496,6 +509,8 @@ export default async function main(progname, args, { env, homedir, agcc }) {
       }
     };
 
+    const swingsetConfig = makeSwingsetConfig(initAction);
```

Comment:
We should probably consume `initAction.resolvedConfig` only at the top of `launchAndInitializeSwingSet`, rather than also this far down.